extensions/firefox/.eslintrc

```diff
@@ -1,8 +1,4 @@
 {
-  "extends": [
-    ../../.eslintrc
```

Comment:
Note: ESLint inherits automatically from eslint config files higher up the tree, so this extends isn't necessary.